tool/gocross/gocross-wrapper.sh

```diff
@@ -26,40 +26,29 @@ repo_root="${BASH_SOURCE%/*}/../.."
 # being invoked from somewhere else.
 cd "$repo_root"
 
-toolchain="$HOME/.cache/tailscale-go"
+# toolchain, set below is the root of the Go toolchain we'll use to build
+# gocross.
+#
+# It's set to either something with a leading slash (if that's what
```

Comment:
What is the "something" in this first case—is that meant to be the path of the toolchain installation? (This may become clear as I read further)

Reply:
reworded for clarity